src/docs/patterns/appbar.mdx

```diff
@@ -83,7 +91,14 @@ A special case of the basic app bar is the dropdown toolbar. This version transf
     />
 </React.Fragment>
 
-In some situations, you may want to present more than just a title and subtitle in an app bar, such as a large graphic, buttons, or multiple lines of text. In these cases, you can use a collapsible app bar that starts out tall but will shrink as the user scrolls down the page until it becomes a basic app bar. This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This is a pattern most commonly used for mobile applications, but could also be used for web.
+Collapsible app bars put a larger emphasize on the app bar content on page load, and provide an opportunity to show images pertain to the page topic.This allows you the flexibility to present an eye-catching banner without permanently reducing the amount of usable space on the page. This pattern is most commonly used for mobile applications, but could also be used for web.
+
+<ImageGrid
+    images={[CollapsibleAppBarDynamic, CollapsibleAppBarSnap]}
+    caption={[
+        `Collapsible app bars have two variants: a dynmaic, scroll-with-finger one that transitions smoothly (first image), and an app bar that "snaps" and collapses once the user scrolls pass a certain point (second image). If you are uncertain whether your page content height is more than one screen tall, use the snap variant to prevent having your app bar stucks halfway through the collapsing.`,
```

Comment:
```suggestion
        `Collapsible app bars can have two slightly different behaviors: 'dynamic', which resizes proportionally to the scroll position (first image), and  'snap', which collapses in one step once the user scrolls pass a certain threshold (second image). You should avoid using collapsible app bars on pages that do not have significant amounts of scrollable content to prevent situations where the app bar can only be partially collapsed or not collapsed at all.`,
```